lib/onc_certification_g10_test_kit.rb

```diff
@@ -74,11 +74,12 @@ class G10CertificationSuite < Inferno::TestSuite
     end
 
     def self.jwks_json
-      @jwks_json ||= File.read(File.join(__dir__, 'onc_certification_g10_test_kit', 'bulk_data_jwks.json'))
+      bulk_data_jwks = JSON.parse(File.read(File.join(__dir__, 'onc_certification_g10_test_kit', 'bulk_data_jwks.json')))
+      @jwks_json ||= { keys: bulk_data_jwks['keys'].select { |key| key['key_ops']&.include?('verify') } }.to_json
```

Comment:
```suggestion
      @jwks_json ||= JSON.pretty_generate({ keys: bulk_data_jwks['keys'].select { |key| key['key_ops']&.include?('verify') } })
```
Small readability fix for the `.well-known` endpoint